sale_payment_term_interest/model/sale_order.py

```diff
@@ -50,26 +50,23 @@ def _prepare_interest_line(self, interest_amount):
 
     @api.multi
     def get_interest_value(self):
+        current_total = 0.
         self.ensure_one()
         term = self.payment_term
         if not term:
             return 0.
         if not any(line.interest_rate for line in term.line_ids):
             return 0.
-        line = self._get_interest_line()
-        if line:
+        for line in [line for line in self.order_line
+                     if not line.interest_line]:
```

Comment:
To keep the modularity you could change the condition to `if not line ==  self._get_interest_line()`